src/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -124,35 +113,23 @@ export interface ConversationProps {
 const Conversation: React.FC<ConversationProps> = props => {
   const { conversation, relay } = props
   return (
-    <>
-      <StyledHeader
-        px={2}
-        alignItems="center"
-        justifyContent="space-between"
-        width="100%"
-      >
-        <RouterLink to={`/user/conversations`}>
-          <ArrowLeftIcon />
-        </RouterLink>
-        <Sans size="3t" weight="medium">
-          Inquiry with {conversation.to.name}
-        </Sans>
-        <InfoCircleIcon />
-      </StyledHeader>
-      <Spacer mt="45px" />
-      <Flex flexDirection="column" width="100%" px={1}>
-        {conversation.items.map((i, idx) => (
-          <Item
-            item={i.item}
-            key={
-              i.item.__typename === "Artwork" || i.item.__typename === "Show"
-                ? i.item.id
-                : idx
-            }
-          />
-        ))}
-        {groupMessages(conversation.messages.edges.map(edge => edge.node)).map(
-          (messageGroup, groupIndex) => {
+    <Flex flexDirection="column" width="100%">
+      <Box>
+        <Spacer mt="45px" />
```

Comment:
@zephraph wouldn't this be better if we did `mt={4}` versus breaking our design system? These values shouldn't be to comp so much as they should be to the closest value in the system. 

Thoughts?

Reply:
This is a buffer to move the conversation body far enough from under a fixed header to be visible. We're moving pretty fast on this and it's not all ideal at the moment. Will follow up.

Reply:
We should just add a `5` to our space scale, this constantly comes up 😄 I'll PR and ping design